spec/index.bs

```diff
@@ -9021,13 +9026,21 @@ The main rendering algorithm:
 
```

Comment:
This is only tangentially related but may as well be in this PR. In "Process vertices":

> An attribute of type "snorm8x2" and byte values of [0x70, 0xD0] will be converted to vec2<f32>(0.88, -0.24) in WGSL.

I think this should this be (0x88, -0.38)? Or perhaps better, (112 &div; 127, -48 &div; 127)